packages/A11yContent/A11yContent.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './A11yContent.modules.scss'
+
+/**
+ * @version ./package.json
+ */
+
+const A11yContent = ({ children }) => {
+  return <p className={styles.a11yContent}>{children}</p>
```

Comment:
I thought since it could be standalone text it should use the `p` tag. Although I'm indifferent to changing it. As for rendering when empty, I think it would. Even when empty, it should still have a place in the DOM. I also think it would be a rare occurrence for it to be provided no children at all.